src/utils/zcl_abapgit_language.clas.xml

```diff
@@ -0,0 +1,49 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<abapGit version="v1.0.0" serializer="LCL_OBJECT_CLAS" serializer_version="v1.0.0">
+ <asx:abap xmlns:asx="http://www.sap.com/abapxml" version="1.0">
+  <asx:values>
+   <VSEOCLASS>
+    <CLSNAME>ZCL_ABAPGIT_LANGUAGE</CLSNAME>
+    <VERSION>1</VERSION>
+    <LANGU>E</LANGU>
+    <EXPOSURE>2</EXPOSURE>
+    <STATE>1</STATE>
+    <CLSCCINCL>X</CLSCCINCL>
+    <FIXPT>X</FIXPT>
+    <UNICODE>X</UNICODE>
+   </VSEOCLASS>
+   <DESCRIPTIONS>
+    <SEOCOMPOTX>
+     <CLSNAME>ZCL_ABAPGIT_LANGUAGE</CLSNAME>
+     <CMPNAME>CLASS_CONSTRUCTOR</CMPNAME>
+     <LANGU>E</LANGU>
+     <DESCRIPT>CLASS_CONSTRUCTOR</DESCRIPT>
+    </SEOCOMPOTX>
+    <SEOCOMPOTX>
+     <CLSNAME>ZCL_ABAPGIT_LANGUAGE</CLSNAME>
+     <CMPNAME>CURRENT_LANGUAGE</CMPNAME>
+     <LANGU>E</LANGU>
+     <DESCRIPT>CURRENT_LANGUAGE</DESCRIPT>
+    </SEOCOMPOTX>
+    <SEOCOMPOTX>
+     <CLSNAME>ZCL_ABAPGIT_LANGUAGE</CLSNAME>
+     <CMPNAME>LOGIN_LANGUAGE</CMPNAME>
+     <LANGU>E</LANGU>
+     <DESCRIPT>LOGIN_LANGUAGE</DESCRIPT>
+    </SEOCOMPOTX>
+    <SEOCOMPOTX>
+     <CLSNAME>ZCL_ABAPGIT_LANGUAGE</CLSNAME>
+     <CMPNAME>RESTORE_LOGIN_LANGUAGE</CMPNAME>
+     <LANGU>E</LANGU>
+     <DESCRIPT>RESTORE_LOGIN_LANGUAGE</DESCRIPT>
+    </SEOCOMPOTX>
+    <SEOCOMPOTX>
+     <CLSNAME>ZCL_ABAPGIT_LANGUAGE</CLSNAME>
+     <CMPNAME>SET_CURRENT_LANGUAGE</CMPNAME>
+     <LANGU>E</LANGU>
+     <DESCRIPT>SET_CURRENT_LANGUAGE</DESCRIPT>
```

Comment:
Perhaps remove the descriptions?

Reply:
Sure, I removed them. Btw. the descpitions were side effects of the import local classes function in SE24.